openslides_backend/shared/env.py

```diff
@@ -1,28 +1,93 @@
-import os
+from enum import Enum, auto
+from typing import Any
 
-from .exceptions import ServerError
-
-INTERNAL_AUTH_PASSWORD_FILE = "INTERNAL_AUTH_PASSWORD_FILE"
-OPENSLIDES_DEVELOPMENT = "OPENSLIDES_DEVELOPMENT"
 DEV_PASSWORD = "openslides"
 
 
+class Loglevel(Enum):
```

Comment:
```suggestion
class Loglevel(str, Enum):
```
will simplify the handling below, since you can then remove the `.name` when accessing members and can create members with `LogLevel(value)` etc.

Reply:
Just tested it and `auto()` seems to always use the index as the value, so in this case `FOO.BAR.value == '1'`. So you would have to either assign the values manually (or use `FOO["BAR"]` instead, but this kinda eliminates the advantage of doing this change in the first place). I think the "double work" by manually assigning values is weighed out by the easier and more intuitive usage.

Reply:
If it is ok for all, I would like to keep the code as is for the moment. The "right" way I think would be usinge a customized Enum class with customized `__getattr__` method. But this is not important at the moment because using `.name` i a few lines does not harm.